java/clients/src/main/java/sleeper/clients/QueryClient.java

```diff
@@ -69,7 +70,6 @@ protected void init(TableProperties tableProperties) throws StateStoreException
         String tableName = tableProperties.get(TABLE_NAME);
         Configuration conf = HadoopConfigurationProvider.getConfigurationForQueryLambdas(getInstanceProperties(), tableProperties);
         conf.setIfUnset("fs.s3a.aws.credentials.provider", DefaultAWSCredentialsProviderChain.class.getName());
```

Comment:
Should this use the Hadoop configuration passed into the constructor rather than the query lambda config?

Reply:
Updated to use the constructor

Reply:
I realise now this has the table properties applied, whereas in the constructor we don't know which table it's for. I guess it needs to be two different configurations, both specific to clients?